### PR TITLE
arm64: dts: crocodile: pmic: set min/max voltages

### PR DESCRIPTION
**Pull request by js731ca**
according to the power (sequence) table found in the baseboard schematic (of rev-b)

"stumbled" over this while upstreaming the imx8mm-revb.dts

applied the changes to barebox too, system seems to boot just fine, and hold up under load(?)
$> md5sum /dev/zero & # times 4
$> cat /proc/loadavg 
4.20 4.20 3.61 5/148 519
=> is there "an appropriate" way to verify those limits?

NOTE: not quite sure about the limits of LDO5 = NVCC_SD2 though...
-> please tripple check (-:

### CODE REVIEW

**Comment by js731ca** [comment]
talked to Patrick, he gave me a set of nominal / "run at" voltagees (not sure where some of those have to go though - if they do at all? - or does the pmic internal voodoo, to set the $correct value, inside the set bounds
he also mentioned that the regulator-min/max are the same as the ones found in the evk-B schematic; and should be ok (?)

i ask myself if adding the nxp,dvs-run-voltages are actually required - since the board "ran fine even without" them?

another question  (maybe @bith3ad ) is: how do these voltage values relate to the "operational performance point" - or the *single* voltage set therein?
since the lpddr seems to need slighly different voltages depending on the frequency

**Comment by js731ca** [comment]
V2:
- checked back with Patrick, limits should be ok; plus /sys/kernel/debug/regulator/regulator_summary reports those values 
- removed 1.6GHz and 1.8GHz OPP, since the crocodile (AFAIK) is supposed to run on the lowest available frequency

**Review by zandrey — APPROVED** [review]

File: arch/arm64/boot/dts/freescale/crocodile.dtsi [hunk 1]
```diff
@@ -139,94 +144,104 @@
...
 				nxp,dvs-standby-voltage = <800000>;
 			};
 
+			// VDD_ARM_0V9
 			buck2_reg: BUCK2 {
 				regulator-name = "BUCK2";
-				regulator-min-microvolt = <600000>;
-				regulator-max-microvolt = <2187500>;
+				regulator-min-microvolt = <805000>;
+				regulator-max-microvolt = <1050000>;
```
Comment (bith3ad) [inline arch/arm64/boot/dts/freescale/crocodile.dtsi:162]: Why all these ranges?

Comment (js731ca) [reply arch/arm64/boot/dts/freescale/crocodile.dtsi:162]: good point made in out-of-band discussion: set the ranges down to the actually desired values -> see PR #113

**Pull request merged**